project.clj

```diff
@@ -15,13 +15,12 @@
                  ;; Confluent does paired releases with Kafka, this should tie
                  ;; off with the kafka version.
                  ;; See https://docs.confluent.io/current/release-notes.html
-                 [io.confluent/kafka-schema-registry-client "5.3.1"
+                 [io.confluent/kafka-schema-registry-client "6.1.1"
                   :exclusions [com.fasterxml.jackson.core/jackson-databind]]
-                 [io.confluent/kafka-avro-serializer "5.3.1"]
-                 [org.apache.kafka/kafka-clients "2.3.1"]
-                 [org.apache.kafka/kafka-streams "2.3.1"]
-                 [org.apache.kafka/kafka-streams-test-utils "2.3.1"]
-
+                 [io.confluent/kafka-avro-serializer "6.1.1"]
+                 [org.apache.kafka/kafka-clients "2.7.0"]
+                 [org.apache.kafka/kafka-streams "2.7.0"]
+                 [org.apache.kafka/kafka-streams-test-utils "2.7.0"]
                  [org.clojure/clojure "1.10.1" :scope "provided"]
                  [org.clojure/data.json "0.2.6"]
                  [org.clojure/data.fressian "0.2.1"]
```

Comment:
@yijiezhen I see Fressian serde was removed, but this dependency still remains. Is there a reason for it?